demo/test.tex

```diff
@@ -0,0 +1,57 @@
+\documentclass{beamer}
+
+% The presentation is supposed to last 30 minutes, starting at 1PM.
+% pdfpc will give a warning for the last 5 minutes of the
+% presentation.
+\usepackage[duration=30,starttime=13:00,lastminutes=5]{../pdfpcnotes}
```

Comment:
I'd prefer to use `pdfpcnotes` here without the path and use TEXINPUTS environment variable in the makefile like here:  https://github.com/cebe/pdfpc-latex-notes/blob/92a6dca9da1557c9305f41885556474b51ed46c4/tests/Makefile#L4

Reply:
I guess essentially my proposal is just another item in your new `tests/` directory. In particular it tests overlays and `\againframe`. So perhaps this PR should be reorganised. Feel free to pick out what you like.

Reply:
I like to have a demo presentation which serves also as part of the documentation. noone would look at tests when trying to figure out how something works.